examples/box2d-examples-window.sln

```diff
@@ -1,72 +1,92 @@
-﻿
```

Comment:
Same here, sorry. Why `box2d-examples-window.sln` was changed? It works by keeping it as is.